#! /bin/sh /usr/share/dpatch/dpatch-run
## g++-4.1_fix.dpatch by Eduard Bloch <blade@debian.org>
## http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=356234

--- cdcat-1.01b~/src/import.h	2005-08-22 19:03:00.000000000 +0200
+++ cdcat-1.01b/src/import.h	2006-03-11 00:23:23.000000000 +0100
@@ -66,7 +66,7 @@
 
 class importGtktalogCsv: public QObject {
 Q_OBJECT public:
-	importGtktalogCsv::importGtktalogCsv( GuiSlave * parent, QString separator, QString filename, bool createdatabase, bool correctbadstyle );
+	importGtktalogCsv( GuiSlave * parent, QString separator, QString filename, bool createdatabase, bool correctbadstyle );
 	~importGtktalogCsv();
 
 	/*

